chevah_format.py

```diff
@@ -0,0 +1,60 @@
+"""
```

Comment:
I think that this can be removed.
This code should be moved to chevah/server and have it executed at runtime